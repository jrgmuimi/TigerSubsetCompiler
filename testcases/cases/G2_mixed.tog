let

in
    printstring("CS 440: Compilers");
    printstring("strings");
    printint(440);
    printstring("testing testing");
    printint(5555555);
    printint(440);
    printstring("long long long string \n");
    printint(1111111);
    printstring("\t\t\t");
    printint(440);
    printstring(" \" escaped chars\" ");
    printint(9999999)
end