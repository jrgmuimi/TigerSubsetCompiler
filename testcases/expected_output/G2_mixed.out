MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

CS 440: Compilersstrings440testing testing5555555440long long long string 
1111111			440 " escaped chars" 9999999
